### PR TITLE
Rename wasUpdated to isEquivalent

### PR DESCRIPTION
**Pull request by kyleve**
Rename wasUpdated to isEquivalent, and invert bool handling... More similar to implementing Equatable, which makes it less surprising.

### CODE REVIEW

File: BlueprintLists/Sources/HeaderFooter.swift [hunk 1]
```diff
@@ -87,9 +87,9 @@ public struct BlueprintHeaderFooterElementAppearance : HeaderFooterElementAppear
     
     public func apply(to view: ContentView) {}
     
-    public func wasUpdated(comparedTo other: BlueprintHeaderFooterElementAppearance) -> Bool
+    public func isIdenticalTo(comparedTo other: BlueprintHeaderFooterElementAppearance) -> Bool
```
Comment (kylebshr) [inline BlueprintLists/Sources/HeaderFooter.swift]: Not to be too nit-picky, but the double "to" reads weirdly to me (`isIdenticalToComparedTo`) - maybe it should be `isIdentical(comparedTo other:)` or `isIdentical(to other:)`? Or even `isIdenticalTo(_ other:)`. Naming is hard.

Comment (kylebshr) [reply BlueprintLists/Sources/HeaderFooter.swift]: FWIW `isEqual(_:)` is how Swift does that but I don't like it

File: Podfile.lock [hunk 2]
```diff
@@ -28,7 +28,7 @@ DEPENDENCIES:
   - Snapshot/Tests (from `Internal Pods/Snapshot/Snapshot.podspec`)
 
 SPEC REPOS:
-  https://github.com/CocoaPods/Specs.git:
+  trunk:
```
Comment (kylebshr) [inline Podfile.lock]: Is this change intentional?

File: BlueprintLists/Sources/HeaderFooter.swift (hunk 1 above)
Comment (kyleve) [reply BlueprintLists/Sources/HeaderFooter.swift]: Ah yeah, good point. Workflow uses `public func isEquivalent(to otherWorker: Self) -> Bool`, which is also not bad. I'll think about it for a bit.

Comment (kyleve) [reply BlueprintLists/Sources/HeaderFooter.swift]: Gonna do `sEquivalent(to:)`.

Comment (kyleve) [reply BlueprintLists/Sources/HeaderFooter.swift]: Pushed!

File: Podfile.lock (hunk 2 above)
Comment (kyleve) [reply Podfile.lock]: This was from updating BlueprintUI – I believe because I havent updated it since I updated Cocoapods, this changes it to pull from the spec CDN, not the repo.
